jf_agent/git/gitlab_adapter.py

```diff
@@ -492,7 +510,7 @@ def _normalize_pr(
         changed_files=changed_files,
         created_at=merge_request.created_at,
         updated_at=merge_request.updated_at,
-        merge_date=merge_request.merged_at,
+        merge_date=merge_request.merge_date,
```

Comment:
Doesn't look right -- wouldn't this break the newer client?

Reply:
This is an example of where I had to move the mapping logic for the v3/v4 gitlab libraries from the gitlab_adapter to the gitlab_v3_client. The reason it changes from `merged_at to merge_date` is because I added that `add_v4_attrs` function to the `gitlab_v3_clients` file.